code/modules/research/xenobiology/crossbreeding/destabilized.dm

```diff
@@ -0,0 +1,322 @@
+			damageToDeal = min(CThrower.get_damage_amount(damageType), 25)
+			CThrower.apply_damage_type(-25, damageType)
+			if(iscarbon(LTarget))
+				var/mob/living/carbon/CTarget = LTarget
+				CTarget.apply_damage_type(damageToDeal, damageType)
+			else
+				LTarget.adjustBruteLoss(damageToDeal)
+		else
+			damageToDeal = -(LThrower.adjustBruteLoss(-25))
+			LTarget.adjustBruteLoss(damageToDeal)
+
+/obj/item/slimecross/destabilized/green/proc/findBiggestLoss(mob/living/carbon/C)
+	var/damageType = BRUTE
+	var/maxDamage = C.get_damage_amount(BRUTE)
+	var/burnDamage = C.get_damage_amount(BURN)
+	if(burnDamage > maxDamage)
+		maxDamage = burnDamage
+		damageType = BURN
+	var/toxinDamage = C.get_damage_amount(TOX)
+	if(toxinDamage > maxDamage)
+		maxDamage = toxinDamage
+		damageType = TOX
+	var/oxyDamage = C.get_damage_amount(OXY)
+	if(oxyDamage > maxDamage)
+		maxDamage = oxyDamage
+		damageType = OXY
+	var/cloneDamage = C.get_damage_amount(CLONE)
+	if(cloneDamage > maxDamage)
+		maxDamage = cloneDamage
+		damageType = CLONE
+	var/staminaDamage = C.get_damage_amount(STAMINA)
+	if(staminaDamage > maxDamage)
+		maxDamage = staminaDamage
+		damageType = STAMINA
+	var/brainDamage = C.get_damage_amount(BRAIN)
+	if(brainDamage > maxDamage)
+		maxDamage = brainDamage
+		damageType = BRAIN
+	return damageType
+
+/obj/item/slimecross/destabilized/pink
+	colour = "pink"
+
+/obj/item/slimecross/destabilized/pink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(!isliving(A) || !iscarbon(thrower))
+		return FALSE //Don't use up the extract unless it lands a hit on a eligible mob.
+	var/mob/living/carbon/CThrower = thrower
+	if(iscarbon(A))
+		var/mob/living/carbon/CTarget = A
+		CTarget.help_shake_act(CThrower)
+	else
+		var/mob/living/LTarget = A
+		var/oldIntent = CThrower.a_intent
+		CThrower.a_intent = "help"
+		LTarget.attack_hand(CThrower)
+		CThrower.a_intent = oldIntent
+
+/obj/item/slimecross/destabilized/gold
+	colour = "gold"
+
+/obj/item/slimecross/destabilized/gold/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	new /obj/effect/temp_visual/goliath_tentacle(get_turf(A), thrower, FALSE)
+
+/obj/item/slimecross/destabilized/oil
+	colour = "oil"
+
+/obj/item/slimecross/destabilized/oil/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	explosion(get_turf(A),-1,-1,2, flame_range = 4)
+
+/obj/item/slimecross/destabilized/black
+	colour = "black"
+
+/obj/item/slimecross/destabilized/black/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.blind_eyes(10)
+		to_chat(L, "<span class='warning'>Your vision is suddenly clouded with inky blackness!</span>")
+	else
+		var/datum/effect_system/smoke_spread/smoke = new
+		smoke.set_up(2, get_turf(A))
+		smoke.start()
+
+/obj/item/slimecross/destabilized/lightpink
+	colour = "light pink"
+
+/obj/item/slimecross/destabilized/lightpink/hitEffect(atom/A, mob/thrower)
+	. = ..()
+	if(isliving(A))
+		var/mob/living/L = A
+		L.reagents.add_reagent("pax", 5)
```

Comment:
We don't use id's for reagents anymore. Please use the typepath for pax instead.